sbin/pack.sh

```diff
@@ -169,11 +159,6 @@ pack_ramp() {
 
 #----------------------------------------------------------------------------------------------
 
-# SEMVER - Semantic version (format: major.minor.patch like 2.6.3)
-# Used for human-readable package naming and release version identification
-SEMVER="$($SBIN/getver)"
```

Comment:
lets delete `sbin/getver` as well

Reply:
deleted no?

Reply:
The file itself is not appearing as deleted